### ISSUE TITLE
UI for coverage property

### ISSUE THREAD

**Issue by grosscol** [mlibrary/umrdr#100]
UI for coverage property needs to be designed & implemented.  Coverage is a date range.  See: http://dublincore.org/documents/dcmi-period/ for details.

Dependent on #99

**Comment by grosscol** [comment]
dependent on #99

**Comment by justcolin** [comment]
@grosscol: is this pretty much a duplicate of #324 (@jonearley's current project)?

**Comment by njaffer** [comment]
As per our conversation today with Matt C, Amy, Jake, Ben and Jon, the UI will look like this:

![img_0658](https://cloud.githubusercontent.com/assets/12487929/14895165/49ef177a-0d45-11e6-93b9-ffa8dcc605a5.JPG)

**Comment by jonearley** [comment]
@njaffer, @amyneeser, @bnhowell

Here is gif showing how the date created field could work with ranges.

![](http://g.recordit.co/PXwWChSoya.gif)

**Comment by jonearley** [comment]
Reopened. I was reading this issue and hit close somehow. Whoops!

**Comment by njaffer** [comment]
done
